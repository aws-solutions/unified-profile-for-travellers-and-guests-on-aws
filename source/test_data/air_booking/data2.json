{    "objectVersion": 19,    "modelVersion": "1.0",    "id": "I2FSJ2",    "externalIds": null,    "lastUpdatedOn": "2023-01-16T07:45:10.919822251Z",    "createdOn": "2022-12-16T07:45:10.919822251Z",    "lastUpdatedBy": "Lindsey Jacobi",    "createdBy": "Mae Schultz",    "creationChannelId": "gds",    "price": {        "status": null,        "grandTotal": 0.0    },    "itinerary": {        "from": "DXB",        "to": "JFK",        "departureDate": "2023-11-03",        "departureTime": "14:58",        "arrivalDate": "2023-11-03",        "arrivalTime": "20:34",        "duration": "5h36m0s",        "segments": [            {                "rank": 0,                "from": "DXB",                "to": "JFK",                "departureDate": "2023-11-03",                "departureTime": "14:58",                "arrivalDate": "2023-11-03",                "arrivalTime": "20:34",                "flightNumber": "UA0044",                "inventory": {                    "totalSeats": 0,                    "inventoryByFareClass": null                },                "status": ""            }        ],        "inventory": {            "totalSeats": 0,            "inventoryByFareClass": null        },        "status": "confirmed"    },    "return": {        "from": "JFK",        "to": "DXB",        "departureDate": "2023-09-13",        "departureTime": "00:04",        "arrivalDate": "2023-09-13",        "arrivalTime": "02:38",        "duration": "2h34m0s",        "segments": [            {                "rank": 0,                "from": "JFK",                "to": "DXB",                "departureDate": "2023-09-13",                "departureTime": "00:04",                "arrivalDate": "2023-09-13",                "arrivalTime": "02:38",                "flightNumber": "UA9803",                "inventory": {                    "totalSeats": 0,                    "inventoryByFareClass": null                },                "status": ""            }        ],        "inventory": {            "totalSeats": 0,            "inventoryByFareClass": null        },        "status": ""    },    "passengerInfo": {        "passengers": [            {                "modelVersion": "1.0",                "id": "5159498203",                "lastUpdatedOn": "2022-07-26T14:26:09.450442124Z",                "createdOn": "2022-06-26T14:26:09.450442124Z",                "lastUpdatedBy": "Obie Skiles",                "createdBy": "Chad Bednar",                "emails": [],                "phones": [                    {                        "type": "business",                        "number": "6430813820",                        "primary": false,                        "countryCode": 12                    },                    {                        "type": "home",                        "number": "7633286117",                        "primary": true,                        "countryCode": 80                    }                ],                "addresses": [],                "honorific": "Dr",                "firstName": "Raphael",                "middleName": "Hector",                "lastName": "Renner",                "gender": "other",                "pronoun": "they",                "dateOfBirth": "1967-02-10",                "language": {                    "code": "sq",                    "name": ""                },                "nationality": {                    "code": "TF",                    "name": ""                },                "jobTitle": "Engineer",                "parentCompany": "The DocGraph Journal",                "loyaltyPrograms": [                    {                        "id": "6707070882",                        "programName": "elite",                        "miles": 152043.0,                        "milesToNextLevel": 1842.0,                        "level": "gold",                        "joined": "2022-06-26T14:26:09.450442124Z"                    },                    {                        "id": "8980415628",                        "programName": "elite",                        "miles": 622557.0,                        "milesToNextLevel": 1719.0,                        "level": "silver",                        "joined": "2022-06-26T14:26:09.450442124Z"                    }                ],                "identityProofs": null            }        ]    },    "paymentInformation": {        "paymentType": "voucher",        "ccInfo": {            "token": "",            "cardType": "",            "cardExp": "",            "cardCvv": "",            "expiration": "",            "name": "",            "address": {                "type": "",                "line1": "",                "line2": "",                "line3": "",                "line4": "",                "city": "",                "state": {                    "code": "",                    "name": ""                },                "province": {                    "code": "",                    "name": ""                },                "postalCode": "",                "country": {                    "code": "",                    "name": ""                },                "primary": false            }        },        "routingNumber": "",        "accountNumber": "",        "voucherID": "IUYHD5AB01HBDZD8",        "address": {            "type": "",            "line1": "",            "line2": "",            "line3": "",            "line4": "",            "city": "",            "state": {                "code": "",                "name": ""            },            "province": {                "code": "",                "name": ""            },            "postalCode": "",            "country": {                "code": "",                "name": ""            },            "primary": false        }    },    "promocode": "VABOVZL98N",    "email": "",    "phone": "",    "status": "tentative"}