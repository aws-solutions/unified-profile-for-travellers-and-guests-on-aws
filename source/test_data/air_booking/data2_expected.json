{"air_booking_recs": [{"model_version": "1.0", "object_type": "air_booking", "last_updated": "2022-04-17T10:59:51.2018075Z", "last_updated_by": "Shana Williamson", "booking_id": "0K6ZGY", "segment_id": "0K6ZGY-DEN-LAX", "from": "DEN", "to": "LAX", "flight_number": "AF1514", "departure_date": "2023-01-02", "departure_time": "17:18", "arrival_date": "2023-01-03", "arrival_time": "05:50", "channel": "web", "status": "canceled", "honorific": "Ms", "first_name": "Kian", "middle_name": "Danyka", "last_name": "Dietrich", "gender": "other", "pronoun": "they", "date_of_birth": "1939-01-18", "job_title": "Technician", "company": "Copyright Clearance Center", "price": 0.0, "nationality_code": "", "nationality_name": "", "language_code": "", "language_name": "", "pss_id": "", "gds_id": "", "payment_type": "loyalty_redemption", "cc_token": "", "cc_type": "", "cc_exp": "", "cc_cvv": "", "cc_name": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_postal_code": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_country": "", "email_business": "", "email": "gudrunprice@swift.net", "email_type": "", "phone": "", "phone_home": "", "phone_mobile": "", "phone_business": "", "traveller_id": "0197775441", "last_updated_partition": "2022/04/17/10/59/51/2018075Z", "address_line1": "", "address_line2": "", "address_line3": "", "address_line4": "", "address_city": "", "address_state_province": "", "address_postal_code": "", "address_country": "", "address_business_line1": "", "address_business_line2": "", "address_business_line3": "", "address_business_line4": "", "address_business_city": "", "address_business_state_province": "", "address_business_postal_code": "", "address_business_country": "", "address_mailing_line1": "", "address_mailing_line2": "", "address_mailing_line3": "", "address_mailing_line4": "", "address_mailing_city": "", "address_mailing_state_province": "", "address_mailing_postal_code": "", "address_mailing_country": "", "address_type": "shipping"}, {"model_version": "1.0", "object_type": "air_booking", "last_updated": "2022-04-17T10:59:51.2018075Z", "last_updated_by": "Shana Williamson", "booking_id": "0K6ZGY", "segment_id": "0K6ZGY-LAX-DEN", "from": "LAX", "to": "DEN", "flight_number": "AF7959", "departure_date": "2022-07-01", "departure_time": "03:07", "arrival_date": "2022-07-01", "arrival_time": "12:21", "channel": "web", "status": "canceled", "honorific": "Ms", "first_name": "Kian", "middle_name": "Danyka", "last_name": "Dietrich", "gender": "other", "pronoun": "they", "date_of_birth": "1939-01-18", "job_title": "Technician", "company": "Copyright Clearance Center", "price": 0.0, "nationality_code": "", "nationality_name": "", "language_code": "", "language_name": "", "pss_id": "", "gds_id": "", "payment_type": "loyalty_redemption", "cc_token": "", "cc_type": "", "cc_exp": "", "cc_cvv": "", "cc_name": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_postal_code": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_country": "", "email_business": "", "email": "gudrunprice@swift.net", "email_type": "", "phone": "", "phone_home": "", "phone_mobile": "", "phone_business": "", "traveller_id": "0197775441", "last_updated_partition": "2022/04/17/10/59/51/2018075Z", "address_line1": "", "address_line2": "", "address_line3": "", "address_line4": "", "address_city": "", "address_state_province": "", "address_postal_code": "", "address_country": "", "address_business_line1": "", "address_business_line2": "", "address_business_line3": "", "address_business_line4": "", "address_business_city": "", "address_business_state_province": "", "address_business_postal_code": "", "address_business_country": "", "address_mailing_line1": "", "address_mailing_line2": "", "address_mailing_line3": "", "address_mailing_line4": "", "address_mailing_city": "", "address_mailing_state_province": "", "address_mailing_postal_code": "", "address_mailing_country": "", "address_type": "shipping"}, {"model_version": "1.0", "object_type": "air_booking", "last_updated": "2022-04-17T10:59:51.2018075Z", "last_updated_by": "Shana Williamson", "booking_id": "0K6ZGY", "segment_id": "0K6ZGY-DEN-LAX", "from": "DEN", "to": "LAX", "flight_number": "AF1514", "departure_date": "2023-01-02", "departure_time": "17:18", "arrival_date": "2023-01-03", "arrival_time": "05:50", "channel": "web", "status": "canceled", "honorific": "Ms", "first_name": "Kiana", "middle_name": "Renee", "last_name": "Harvey", "gender": "other", "pronoun": "he", "date_of_birth": "1960-03-06", "job_title": "Liaison", "company": "Wolters Kluwer", "price": 0.0, "nationality_code": "", "nationality_name": "", "language_code": "", "language_name": "", "pss_id": "", "gds_id": "", "payment_type": "loyalty_redemption", "cc_token": "", "cc_type": "", "cc_exp": "", "cc_cvv": "", "cc_name": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_postal_code": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_country": "", "email_business": "", "email": "seamusschamberger@bode.biz", "email_type": "", "phone": "1-094-970-4457", "phone_home": "", "phone_mobile": "", "phone_business": "", "phone_type": "", "traveller_id": "7478240392", "last_updated_partition": "2022/04/17/10/59/51/2018075Z", "address_line1": "4593 Ville town", "address_line2": "more content line 2", "address_line3": "more content line 3", "address_line4": "more content line 4", "address_city": "Austin", "address_state_province": "", "address_postal_code": "84983", "address_country": "VA", "address_business_line1": "", "address_business_line2": "", "address_business_line3": "", "address_business_line4": "", "address_business_city": "", "address_business_state_province": "", "address_business_postal_code": "", "address_business_country": "", "address_mailing_line1": "", "address_mailing_line2": "", "address_mailing_line3": "", "address_mailing_line4": "", "address_mailing_city": "", "address_mailing_state_province": "", "address_mailing_postal_code": "", "address_mailing_country": "", "address_type": ""}, {"model_version": "1.0", "object_type": "air_booking", "last_updated": "2022-04-17T10:59:51.2018075Z", "last_updated_by": "Shana Williamson", "booking_id": "0K6ZGY", "segment_id": "0K6ZGY-LAX-DEN", "from": "LAX", "to": "DEN", "flight_number": "AF7959", "departure_date": "2022-07-01", "departure_time": "03:07", "arrival_date": "2022-07-01", "arrival_time": "12:21", "channel": "web", "status": "canceled", "honorific": "Ms", "first_name": "Kiana", "middle_name": "Renee", "last_name": "Harvey", "gender": "other", "pronoun": "he", "date_of_birth": "1960-03-06", "job_title": "Liaison", "company": "Wolters Kluwer", "price": 0.0, "nationality_code": "", "nationality_name": "", "language_code": "", "language_name": "", "pss_id": "", "gds_id": "", "payment_type": "loyalty_redemption", "cc_token": "", "cc_type": "", "cc_exp": "", "cc_cvv": "", "cc_name": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_postal_code": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_country": "", "email_business": "", "email": "seamusschamberger@bode.biz", "email_type": "", "phone": "1-094-970-4457", "phone_home": "", "phone_mobile": "", "phone_business": "", "phone_type": "", "traveller_id": "7478240392", "last_updated_partition": "2022/04/17/10/59/51/2018075Z", "address_line1": "4593 Ville town", "address_line2": "more content line 2", "address_line3": "more content line 3", "address_line4": "more content line 4", "address_city": "Austin", "address_state_province": "", "address_postal_code": "84983", "address_country": "VA", "address_business_line1": "", "address_business_line2": "", "address_business_line3": "", "address_business_line4": "", "address_business_city": "", "address_business_state_province": "", "address_business_postal_code": "", "address_business_country": "", "address_mailing_line1": "", "address_mailing_line2": "", "address_mailing_line3": "", "address_mailing_line4": "", "address_mailing_city": "", "address_mailing_state_province": "", "address_mailing_postal_code": "", "address_mailing_country": "", "address_type": ""}, {"model_version": "1.0", "object_type": "air_booking", "last_updated": "2022-04-17T10:59:51.2018075Z", "last_updated_by": "Shana Williamson", "booking_id": "0K6ZGY", "segment_id": "0K6ZGY-DEN-LAX", "from": "DEN", "to": "LAX", "flight_number": "AF1514", "departure_date": "2023-01-02", "departure_time": "17:18", "arrival_date": "2023-01-03", "arrival_time": "05:50", "channel": "web", "status": "canceled", "honorific": "Ms", "first_name": "Kian", "middle_name": "Danyka", "last_name": "Dietrich", "gender": "other", "pronoun": "they", "date_of_birth": "1939-01-18", "job_title": "Technician", "company": "Copyright Clearance Center", "price": 0.0, "nationality_code": "", "nationality_name": "", "language_code": "", "language_name": "", "pss_id": "", "gds_id": "", "payment_type": "loyalty_redemption", "cc_token": "", "cc_type": "", "cc_exp": "", "cc_cvv": "", "cc_name": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_postal_code": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_country": "", "email_business": "", "email": "gudrunprice@swift.net", "email_type": "", "phone": "", "phone_home": "", "phone_mobile": "", "phone_business": "", "traveller_id": "0197775441", "last_updated_partition": "2022/04/17/10/59/51/2018075Z", "address_line1": "", "address_line2": "", "address_line3": "", "address_line4": "", "address_city": "", "address_state_province": "", "address_postal_code": "", "address_country": "", "address_business_line1": "", "address_business_line2": "", "address_business_line3": "", "address_business_line4": "", "address_business_city": "", "address_business_state_province": "", "address_business_postal_code": "", "address_business_country": "", "address_mailing_line1": "", "address_mailing_line2": "", "address_mailing_line3": "", "address_mailing_line4": "", "address_mailing_city": "", "address_mailing_state_province": "", "address_mailing_postal_code": "", "address_mailing_country": "", "address_type": "shipping"}, {"model_version": "1.0", "object_type": "air_booking", "last_updated": "2022-04-17T10:59:51.2018075Z", "last_updated_by": "Shana Williamson", "booking_id": "0K6ZGY", "segment_id": "0K6ZGY-LAX-DEN", "from": "LAX", "to": "DEN", "flight_number": "AF7959", "departure_date": "2022-07-01", "departure_time": "03:07", "arrival_date": "2022-07-01", "arrival_time": "12:21", "channel": "web", "status": "canceled", "honorific": "Ms", "first_name": "Kian", "middle_name": "Danyka", "last_name": "Dietrich", "gender": "other", "pronoun": "they", "date_of_birth": "1939-01-18", "job_title": "Technician", "company": "Copyright Clearance Center", "price": 0.0, "nationality_code": "", "nationality_name": "", "language_code": "", "language_name": "", "pss_id": "", "gds_id": "", "payment_type": "loyalty_redemption", "cc_token": "", "cc_type": "", "cc_exp": "", "cc_cvv": "", "cc_name": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_postal_code": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_country": "", "email_business": "", "email": "gudrunprice@swift.net", "email_type": "", "phone": "", "phone_home": "", "phone_mobile": "", "phone_business": "", "traveller_id": "0197775441", "last_updated_partition": "2022/04/17/10/59/51/2018075Z", "address_line1": "", "address_line2": "", "address_line3": "", "address_line4": "", "address_city": "", "address_state_province": "", "address_postal_code": "", "address_country": "", "address_business_line1": "", "address_business_line2": "", "address_business_line3": "", "address_business_line4": "", "address_business_city": "", "address_business_state_province": "", "address_business_postal_code": "", "address_business_country": "", "address_mailing_line1": "", "address_mailing_line2": "", "address_mailing_line3": "", "address_mailing_line4": "", "address_mailing_city": "", "address_mailing_state_province": "", "address_mailing_postal_code": "", "address_mailing_country": "", "address_type": "shipping"}], "common_email_recs": [{"model_version": "1.0", "object_type": "email_history", "last_updated": "2022-04-17T10:59:51.2018075Z", "last_updated_by": "Shana Williamson", "address": "gudrunprice@swift.net", "type": "", "traveller_id": "0197775441", "last_updated_partition": "2022/04/17/10/59/51/2018075Z"}, {"model_version": "1.0", "object_type": "email_history", "last_updated": "2022-04-17T10:59:51.2018075Z", "last_updated_by": "Shana Williamson", "address": "mireyawillms@kuphal.biz", "type": "", "traveller_id": "0197775441", "last_updated_partition": "2022/04/17/10/59/51/2018075Z"}, {"model_version": "1.0", "object_type": "email_history", "last_updated": "2022-04-17T10:59:51.2018075Z", "last_updated_by": "Shana Williamson", "address": "seamusschamberger@bode.biz", "type": "", "traveller_id": "7478240392", "last_updated_partition": "2022/04/17/10/59/51/2018075Z"}, {"model_version": "1.0", "object_type": "email_history", "last_updated": "2022-04-17T10:59:51.2018075Z", "last_updated_by": "Shana Williamson", "address": "gudrunprice@swift.net", "type": "", "traveller_id": "0197775441", "last_updated_partition": "2022/04/17/10/59/51/2018075Z"}, {"model_version": "1.0", "object_type": "email_history", "last_updated": "2022-04-17T10:59:51.2018075Z", "last_updated_by": "Shana Williamson", "address": "mireyawillms@kuphal.biz", "type": "", "traveller_id": "0197775441", "last_updated_partition": "2022/04/17/10/59/51/2018075Z"}], "common_phone_recs": [{"model_version": "1.0", "object_type": "phone_history", "last_updated": "2022-04-17T10:59:51.2018075Z", "last_updated_by": "Shana Williamson", "number": "1-094-970-4457", "country_code": 52, "type": "", "traveller_id": "7478240392", "last_updated_partition": "2022/04/17/10/59/51/2018075Z"}], "air_loyalty_recs": [{"model_version": "1.0", "object_type": "air_loyalty", "last_updated": "2022-04-17T10:59:51.2018075Z", "last_updated_by": "Shana Williamson", "id": "1726659371", "program_name": "elite", "miles": 314504.0, "miles_to_next_level": 4831.0, "level": "silver", "joined": "2021-08-16T00:16:33.420688138Z", "traveller_id": "0197775441", "last_updated_partition": "2022/04/17/10/59/51/2018075Z"}, {"model_version": "1.0", "object_type": "air_loyalty", "last_updated": "2022-04-17T10:59:51.2018075Z", "last_updated_by": "Shana Williamson", "id": "5978089185", "program_name": "club", "miles": 586674.0, "miles_to_next_level": 2109.0, "level": "bronze", "joined": "2021-08-16T00:16:33.420688138Z", "traveller_id": "0197775441", "last_updated_partition": "2022/04/17/10/59/51/2018075Z"}, {"model_version": "1.0", "object_type": "air_loyalty", "last_updated": "2022-04-17T10:59:51.2018075Z", "last_updated_by": "Shana Williamson", "id": "6138593161", "program_name": "ascend", "miles": 773929.0, "miles_to_next_level": 130.0, "level": "bronze", "joined": "2022-01-14T05:18:24.148752386Z", "traveller_id": "7478240392", "last_updated_partition": "2022/04/17/10/59/51/2018075Z"}, {"model_version": "1.0", "object_type": "air_loyalty", "last_updated": "2022-04-17T10:59:51.2018075Z", "last_updated_by": "Shana Williamson", "id": "1726659371", "program_name": "elite", "miles": 314504.0, "miles_to_next_level": 4831.0, "level": "silver", "joined": "2021-08-16T00:16:33.420688138Z", "traveller_id": "0197775441", "last_updated_partition": "2022/04/17/10/59/51/2018075Z"}, {"model_version": "1.0", "object_type": "air_loyalty", "last_updated": "2022-04-17T10:59:51.2018075Z", "last_updated_by": "Shana Williamson", "id": "5978089185", "program_name": "club", "miles": 586674.0, "miles_to_next_level": 2109.0, "level": "bronze", "joined": "2021-08-16T00:16:33.420688138Z", "traveller_id": "0197775441", "last_updated_partition": "2022/04/17/10/59/51/2018075Z"}]}